dwdatareader/__init__.py

```diff
@@ -1,171 +1,347 @@
-"""Python module to interact with Dewesoft DWDataReaderLib.dll
+"""Python module that wraps Dewesoft DWDataReaderLib.dll for interactive use with Pyton
 
-@author: shelmore and costerwisch
+@author: shelmore, costerwisch and Bojan023
```

Comment:
@costerwi should we split this into `maintainers` and `contributors`?

Reply:
```suggestion
Homepage https://github.com/costerwi/dwdatareader
```